vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1102,6 +1109,7 @@ def torch_vllm_outplace_fused_experts(**kwargs) -> torch.Tensor:
 
 def dispatch_fused_experts_func(inplace: bool) -> Callable[..., torch.Tensor]:
     if is_rocm_aiter_moe_enabled():
+        # TODO: support mxfp4 with aiter?
```

Comment:
Is it possible to use mxfp4 in triton?

Reply:
yes, kernel bring up will be in a separate follow up PR.